koku/api/user_access/view.py

```diff
@@ -28,121 +28,88 @@ class UIFeatureAccess:
         1. to keep UI and API in sync about RBAC permissions
         2. to provide the UI (both our UI and the platform's chroming) with a simple boolean response regarding whether
             a user has access to specific features of the UI.
-
-    Class attributes:
-
-        access_keys (list) - a list of access keys to check. must be implemented by sub-classes.
-                             multiple keys will be ORed; i.e. if a user has access to anything in the list,
-                             this API returns True
     """
 
-    def __init__(self, access):
+    PRERELEASE = ["ibm"]
+
+    def __init__(self, access, admin_user, access_key_list=None):
```

Comment:
Where do we pass `access_key_list` into this init?

Reply:
I missed this in my clean up 🙇